osquery/tables/events/linux/process_events.cpp

```diff
@@ -278,7 +278,10 @@ Status AuditProcessEventSubscriber::ProcessExecveEventData(
       row["cmdline"] += ' ';
     }
 
+    // Shell syntax, use to prevent spaces from changing semantics
+    row["cmdline"] += "'";
     row["cmdline"] += DecodeAuditPathValues(arg.second);
```

Comment:
This function should probably only add quotes when they are needed due to spaces in the argument. Additionally, it would be nice to escape inner quotes if found, but I defer this decision to @theopolis